src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {

```

Comment:
The class `Figure` is using the default access modifier. Consider using `public` if this class is meant to be accessed from other packages.